scripts/icu/63.1-min-static-data/script.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+
+# Build ICU common package (libicuuc.a) with data file separate and with support for legacy conversion and break iteration turned off in order to minimize size
+
+MASON_NAME=icu
+MASON_VERSION=63.1-min-static-data
+MASON_LIB_FILE=lib/libicuuc.a
+#MASON_PKGCONFIG_FILE=lib/pkgconfig/icu-uc.pc
+
+. ${MASON_DIR}/mason.sh
+
+MASON_BUILD_DEBUG=0 # Enable to build library with debug symbols
+MASON_CROSS_BUILD=0
+
+function mason_load_source {
+    mason_download \
+        http://download.icu-project.org/files/icu4c/63.1/icu4c-63_1-src.tgz \
+        3e24e6839878321f5193a3c4aa6f02210052fb31 \
+        source
+
+    mason_extract_tar_gz
+
+    mason_download \
+        http://download.icu-project.org/files/icu4c/63.1/icu4c-63_1-data.zip \
+        a0eb02a525822ea5ab409e04cf280bacccbb5d4d \
+        data
+
+    mason_extract_zip -d ${MASON_NAME}/source
```

Comment:
prior to the `SLUG_SUFFIX` this function would have zapped the cache for the previous download.